test/integration/APIClient.test.ts

```diff
@@ -126,7 +126,7 @@ describe('APIClient() with chain settings', () => {
 
   describe('#verifyProofForBlock', () => {
     it('expect to return valid result', async () => {
-      const verificationResult = await apiClient.verifyProofForNewestBlock('UMB-USD');
+      const verificationResult = await apiClient.verifyProofForNewestBlock('ADA-USD');
```

Comment:
hmmm why we want to use some other token? I used `UMB` because it is our token, not sure how it is possible that this test start failing.

Reply:
sometimes the UMB-USD is not in the block. Since the verifyProof use the lastest block it fails when UMB-USD is not there.

Reply:
well, it might be same stuff, but... if this will cause issues, we can write more suffisticated test to check maybe 2-3 block behind.

Reply:
Since it is the latestBlock I don't think I can retry in a different blockId. I just can do it if I create a new method in APIClient, but I don't think I should do that.